### ISSUE TITLE
Intro vanishes on Neovim Startup

### ISSUE THREAD

**Issue by nc1337** [glepnir/galaxyline.nvim#198]
Neovim intro splash screen vanishes after a second or so

https://user-images.githubusercontent.com/10063565/125589904-28b51a17-3bd7-4af8-a3a2-65a03ee74a3d.mov

After removing the plugin, splash screen comes up:

![image](https://user-images.githubusercontent.com/10063565/125589620-1107078f-60a2-4e88-b193-d1950b3cbafa.png)

**Comment by krzykli** [comment]
Yeah I'm having the exact same issue with 0.5.0 release. I'm also on a mac.
Not sure what's going on, galaxyline doesn't seem to appear at all. No errors, no nothing, fails silently somewhere 🤔 
Thanks for looking into it!
